charts/console/Chart.yaml

```diff
@@ -3,7 +3,7 @@ name: console
 description: >-
   deploys the plural console and additional dependencies, for use in bring-your-own-kube setups
 appVersion: 0.11.2
-version: 0.3.79
+version: 0.3.80
```

Comment:
let's wait until release to bump this (don't want any chart versions published w/o e2e testing)